web/packages/teleterm/src/main.ts

```diff
@@ -239,6 +245,10 @@ function initializeApp(): void {
         logger.warn(
           `Opening a new window to ${url} blocked by 'setWindowOpenHandler'`
         );
+        dialog.showErrorBox(
+          'Cannot open this link',
+          'The domain this link points to does not match any of the allowed domains. Check main.log for more details.'
```

Comment:
WDYT of simplifying this message? We already have "this link" in the title, so I think we can skip it later.
>The domain does not match any of the allowed domains. Check main.log for more details.